rules/windows/credential_access_shadowcopy_via_mklink.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/12/25"
+maturity = "production"
+updated_date = "2021/12/25"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when shadow copies storage symbolic link is created.
+"""
+false_positives = ["System Administrator working with shadow copies"]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"] 
+language = "eql"
+license = "Elastic License v2"
+name = "Shadowcopy via mklink"
+references = [
+    "https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/mklink",
+    "https://2017.zeronights.org/wp-content/uploads/materials/ZN17_Kheirkhabarov_Hunting_for_Credentials_Dumping_in_Windows_Environment.pdf",
+]
+risk_score = 47
+rule_id = "d117cbb4-7d56-41b4-b999-bdf8c25648a0"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+process.pe.original_file_name == "cmd.exe" and
+process.args : "*mklink*" and
+process.args : "*\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy*" and
+process.args : "/C" and
+process.args : "/D"
```

Comment:
```suggestion
(process.args : "/D" or process.command_line : "*/D *")
```

Reply:
The /c is from cmd, and using the "/D" alone will not alert when the command is put between quotes, so let's change it to this by now